core-services/prow/02_config/generate-boskos.py

```diff
@@ -64,7 +64,8 @@
         'usgovvirginia': 5
     },
     'azure-qe-quota-slice': {
-        'northcentralus': 5
+        'northcentralus': 5,
+        'southcentralus': 2
```

Comment:
I prefer untouched with this slice, in the future, if we need more quota after migrate testing from jenkins ci to prow ci, we will enlarge the quota on that time.

Reply:
I need add southcentralus to fix "{  failed to acquire lease for "azure-qe-quota-slice": resources not found}"

Reply:
I think all the quota was occupied on that time, I suggest to wait some time to retest for a luck. So far, we do not want to prow ci occupy too much azure quota before we move jenkins ci workload to prow ci.